reframe/frontend/loader.py

```diff
@@ -5,19 +5,17 @@
 import ast
 import collections.abc
 import os
-import logging
-import sys
+from importlib.machinery import SourceFileLoader
+
 import reframe.core.debug as debug
 import reframe.utility.os as os_ext
-
-from importlib.machinery import SourceFileLoader
+# Pycharm detects the import of ProgEnvironment wrongly as unneeded.
```

Comment:
The source of this problem is further down the code. I'll do a PR into your branch to make PyCharm happy too!